recipes-core/lcd-tools/lcd-tools/lcd-sync-time.cpp

```diff
@@ -0,0 +1,28 @@
+#include "lcd-tools.h"
+
+#include <iostream>
+#include <dlfcn.h>
+#include <hybris/common/dlfcn.h>
+
+int AsteroidOS::LCD_Tools::SyncTime() {
+	void* handler;
+	int (*syncTime)();
```

Comment:
Ugh this is no longer C89, you don't have to define local variables as a preambles. It would be a lot more readable to declare them where they get first assigned